cinema/models.py

```diff
@@ -84,23 +84,39 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
+    @staticmethod
+    def validate_ticket(
+            row: int,
+            seat: int,
+            movie_session: MovieSession,
+            error_to_raise
```

Comment:
The `error_to_raise` parameter is expected to be a callable exception class. Ensure that it is always passed a valid exception class that can be instantiated with a dictionary of errors. Consider adding a check to ensure `error_to_raise` is callable before using it.